src/interfaces/IAccount_NEW.sol

```diff
@@ -42,4 +42,15 @@ interface IAccount_NEW {
     function accountManagementAction(address actionHandler, bytes calldata actionData)
         external
         returns (address, uint256);
+
+    function checkAndStartLiquidation()
```

Comment:
This is a view function right?

Reply:
As of now, no, the account will get into the inAuction state by changing variable value in it. Until we are %100 sure that the Account Owner cannot do anything malicious during the auction, this cannot be view.